src/components/calculator-screen.js

```diff
@@ -0,0 +1,8 @@
+function calculatorScreen(screenId,inputId) {
+    return`
```

Comment:
Hey @Willian1661 

I was thinking,  as this screen input gonna be Update dynamically I guess it will be quite hard to keep this into the component.

Then let's create the calculator component  :)
You don't need this screen component ATM  ;)

Reply:
> so i change this PR for the whole calculator body? already with the screen and the buttons?

yep